packages/manifest/src/index.ts

```diff
@@ -0,0 +1,85 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { TeamsAppManifest, IStaticTab, IConfigurableTab } from "@microsoft/teamsfx-api";
+import * as fs from "fs-extra";
+
+export class Manifest {
+  private manifest: TeamsAppManifest;
```

Comment:
Should this really be `private`?
I think there is value in providing a way to access the `TeamsAppManifest`. Having some `get` function at least would be nice

Reply:
Good point. It seems you want to directly access `TeamsAppManifest`. How about instead of adding another layer of indirection, we just re-export the `TeamsAppManifest` type, and provide a few functions to work with it. e.g.
`loadFromPath(path: string): Promise<TeamsAppManifest>`
`save(manifest: TeamsAppManifest, path: string): Promise<void>`
`validate(manifest: TeamsAppManifest): Promise<string[]>`

Reply:
@igor-ribeiiro Are there any other functions you feel like having?

Reply:
Not for now, those seem great!